build.sbt

```diff
@@ -29,6 +29,7 @@ libraryDependencies ++= Seq(
   "org.squbs" %% "squbs-actorregistry" % squbsV,
   "org.squbs" %% "squbs-httpclient" % squbsV,
   "org.squbs" %% "squbs-admin" % squbsV,
+  "org.squbs" %% "squbs-ext" % squbsV,
```

Comment:
I do not think we need this explicitly.  `squbs-httpclient` should already be bringing it.